src/conda_project/project.py

```diff
@@ -804,9 +805,8 @@ def clean(
         verbose: bool = False,
     ) -> None:
         """Remove the conda environment."""
-
         _ = call_conda(
-            ["env", "remove", "-p", str(self.prefix)],
+            ["env", "remove", "-y", "-p", str(self.prefix)],
```

Comment:
I changed this because there was a test failure, but it doesn't seem related to the changes in this PR, as the test also failed on main when I tried it locally. Not sure when the test started failing but passing `-y` seems like the correct thing to do here in any case.

Reply:
👍